### ISSUE TITLE
Sourcemaps?

### ISSUE THREAD

**Issue by jeffgca** [bradleyboy/yarsk#12]
For browsers / devtools like Chrome and Firefox that support sourcemaps, it would be great to have generated sourcemaps.

**Comment by bradleyboy** [comment]
@canuckistani Yeah, evidently I need to document this better :smiley:. In the meantime, here's a solution: https://github.com/bradleyboy/yarsk/issues/10#issuecomment-91746690
